gamification/src/Components/EditableListItem/EditableListItem.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function EditableListItem() {
```

Comment:
asign todo to @FanchiniRudolf

Reply:
Assigning to your username as stated on your comment. Not sure if it has some effect or if I could type any name.